browser/lib/markdown.js

```diff
@@ -76,7 +76,16 @@ md.use(require('markdown-it-named-headers'), {
   }
 })
 md.use(require('markdown-it-kbd'))
-md.use(require('markdown-it-plantuml'))
+md.use(require('markdown-it-plantuml'), '', {
+  generateSource: function (umlCode) {
+    var deflate = require('markdown-it-plantuml/lib/deflate')
```

Comment:
pleze do not use dynamic import. `.mjs` don't support dynamic import.(because `.mjs` loading is async)

Reply:
what do you mean by .mjs? The deflate script is just a normal .js file...

Reply:
`.mjs` means next feature of ecmascript.
fyi -> https://docs.google.com/presentation/d/1JKnj-HvAwkba9GMEuEqZjMPzAaZTL7j2f1n4JjOWXhs/edit#slide=id.p